mlClassifier/app.py

```diff
@@ -1,14 +1,104 @@
-from flask import Flask
-import os
+import http
+
+from Base_Learner.SHModelUtils import *
+from flask import Flask, Response, request
+import os, werkzeug
+import requests
 
 app = Flask(__name__)
 
+types = [PYTHON3_LANG_NAME, KOTLIN_LANG_NAME, JAVA_LANG_NAME, "python"]
+
+# Helper method to make testing easier
+def getLexing(text, type):
```

Comment:
I took the highlight and lexing call out of the functions below. This makes mocking them for easier. @alainkueng @epura